doc/storageprofile.md

```diff
@@ -36,19 +36,28 @@ status:
     cloneStrategy: snapshot
 ```
 
-Current version supports the following parameters:
+### Parameters
 - `cloneStrategy` - defines the preferred method for performing a CDI clone
 - `claimPropertySets` contains a list of `claimPropertySet`
   - `accessMode` - contains the desired access modes the volume should have
-  - `volumeMode` - defines what type of volume is required by the claim
+  - `volumeMode` - defines what type of volume is required by the claim  
+  These are ordered by preference, so some things to note are:  
+  &emsp;&emsp;Usually Block is preferred over Filesystem for performance reasons (fewer layers)  
+  &emsp;&emsp;RWX over RWO (live migration support).
+- `dataImportCronSourceFormat` DataImportCron (recurring polling of golden registry sources) was originally designed to only maintain PVC sources, However, for certain storage types, we know that snapshots sources scale better. Some details and examples can be found in [clone-from-volumesnapshot-source](./clone-from-volumesnapshot-source.md).
 
 Values for accessModes and volumeMode are exactly the same as for PVC: `accessModes` is a list of `[ReadWriteMany|ReadWriteOnce|ReadOnlyMany]`
 and `volumeMode` is a single value `Filesystem` or `Block`.
 Multiple claim property sets can be specified (`claimPropertySets` is a list).
 
-The value for `cloneStrategy` can be one of: `copy`,`snapshot`,`csi-clone`. 
+The value for `cloneStrategy` can be one of:
+- `copy` - copy bytes over network
```

Comment:
Is "bytes" the best terminology or should we say "copy blocks of data over the network"